frontend/components/common/ControlledInput.tsx

```diff
@@ -30,20 +26,23 @@ export function ControlledInput<TFieldValues extends FieldValues>({
       <Text fontSize="small" fontWeight="600">
         {label}
       </Text>
-      <Input
-        type={type}
-        size="sm"
-        borderRadius="5px"
-        {...field}
-        backgroundColor="white"
-      />
-      {error?.message && (
-        <span
-          className={clsx(
-            'text-red-900 bg-red-400 absolute bottom-[40px] right-0 rounded-md px-2 py-1 text-[10px]',
-            'after:bg-red-400 after:absolute after:-bottom-1 after:right-2 after:h-2 after:w-2 after:rotate-45',
```

Comment:
It is wired to delele this after part 
![image](https://github.com/user-attachments/assets/467ff482-e8e2-4111-8d5e-b99aacdc35e7)